src/main/scala/scorex/core/network/NetworkController.scala

```diff
@@ -201,6 +234,27 @@ class NetworkController(settings: NetworkSettings,
     }
   }
 
+  /**
+    * Schedule a periodic dropping of connections which seem to be inactive
+    */
+  private def scheduleDroppingDeadConnections(): Unit = {
+    context.system.scheduler.schedule(60.seconds, 60.seconds) {
+      // Drop connections with peers if they seem to be inactive
+      val now = networkTime()
+      connections.values.foreach { cp =>
+        val lastSeen = cp.peerInfo.map(_.lastSeen).getOrElse(now)
+        // A peer should send out sync message to us at least once per settings.syncStatusRefreshStable duration.
+        // We wait for more, namely settings.syncStatusRefreshStable.toMillis * 2
+        val timeout = settings.syncStatusRefreshStable.toMillis * 2
```

Comment:
I suggest to increase this multiple to 3 or maybe even 4. To avoid potential problems with loosing connectivity. It much less comparing to the current non-closing behavior (kind of infinite timeout).

Reply:
changed the factor to 3